backend/src/model/entity.rs

```diff
@@ -11,70 +11,230 @@ use diesel::{Identifiable, Insertable, Queryable};
 use crate::schema::{plants, seeds};
 
 use super::r#enum::{
-    deciduous_or_evergreen::DeciduousOrEvergreen, fertility::Fertility, flower_type::FlowerType,
-    growth_rate::GrowthRate, herbaceous_or_woody::HerbaceousOrWoody, life_cycle::LifeCycle,
-    nutrition_demand::NutritionDemand, quality::Quality, quantity::Quantity,
-    root_zone_tendancy::RootZoneTendancy, soil_ph::SoilPh, soil_texture::SoilTexture,
-    soil_water_retention::SoilWaterRetention, sun::Sun, water::Water,
+    deciduous_or_evergreen::DeciduousOrEvergreen, external_source::ExternalSource,
+    fertility::Fertility, flower_type::FlowerType, growth_rate::GrowthRate,
+    herbaceous_or_woody::HerbaceousOrWoody, life_cycle::LifeCycle,
+    light_requirement::LightRequirement, nutrition_demand::NutritionDemand,
+    propagation_method::PropagationMethod, quality::Quality, quantity::Quantity, soil_ph::SoilPh,
+    soil_texture::SoilTexture, soil_water_retention::SoilWaterRetention,
+    water_requirement::WaterRequirement,
 };
 
 /// The `Plants` entity.
 #[allow(clippy::missing_docs_in_private_items)] // TODO: See #97.
 #[derive(Identifiable, Queryable)]
 #[diesel(table_name = plants)]
 pub struct Plants {
+    /// The internal id of the plant.
     pub id: i32,
-    pub binomial_name: String,
-    pub common_name: Option<Vec<Option<String>>>,
+    /// The unique name of the plant. The structure is described in the /doc/database/hierarchy.md document in the repository.
+    pub unique_name: String,
+    /// The list of the common names of the plant.
+    pub common_name_en: Option<Vec<Option<String>>>,
+    /// The list of the common names of the plant in german. Fetched from Wikidata API if not present in the source datasets.
```

Comment:
I would prefer if they are always fetched (and added to the array if not already there).

```suggestion
    /// The list of the common names of the plant in German. Fetched from Wikidata API if not present in the source datasets.
```